src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -116,61 +112,35 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
     "a user exists"
   )
   fun `a user exists`() {
-    userDetailsRepository.save(
-      UserDetails(
-        userId,
-        FirstName("Bertie"),
-        LastName("Badger"),
-        "",
-        Email("b@b.com"),
-        PhoneNumber("0987654321"),
-        CaseworkerBand.FOUR_PLUS,
-        OffsetDateTime.now()
-      )
-    )
+    setupUserDetailsFor(userIdOnes)
   }
 
   @State(
     "a user and a fully populated recall without documents exists"
   )
   fun `a user and a fully populated recall without documents exists`() {
-    val assigneeUuid = UUID.fromString("11111111-1111-1111-1111-111111111111")
-    val recall = fullyPopulatedRecall(::RecallId.zeroes()).copy(
+    val recall = fullyPopulatedRecall(::RecallId.zeroes(), userIdOnes).copy(
       documents = emptySet(),
       missingDocumentsRecords = emptySet(),
-      assignee = assigneeUuid
-    )
-    userDetailsRepository.save(
-      UserDetails(
-        UserId(assigneeUuid), FirstName("Bertie"), LastName("Badger"), "", Email("b@b.com"), PhoneNumber("0987654321"),
-        CaseworkerBand.FOUR_PLUS,
-        OffsetDateTime.now()
-      )
+      assignee = userIdOnes.value
     )
+    setupUserDetailsFor(userIdOnes)
+    setupUserDetailsFor(::UserId.zeroes())
```

Comment:
? `::UserId.zeroes()` is consistent with `userIdOnes `?  I just mean: they could be created in the same way as one another 'tis all.  V. minor I guess -> later tidying.

Reply:
I couldn't get it to work!